src/hoc/selectable-enhance.js

```diff
@@ -65,7 +65,7 @@ export const SelectableContainerEnhance = (Component) => {
         };
       }
 
-      listItems = React.Children.map(children, (child) => {
+      function extendChild(child) {
```

Comment:
If we can, could you move this method outside the render method?
That's going to be better for perf. We don't want this to be allocated as each rendering.